Dockerfile

```
@@ -2,7 +2,7 @@ ARG BASE_IMAGE=nansencenter/geospaas
 
 FROM ${BASE_IMAGE} as base
 
-ARG METANORM_VERSION=1.0.0
+ARG METANORM_VERSION=1.0.1
```

Can you remove the default version? It's confusing to have the metanorm version in two places
```suggestion
ARG METANORM_VERSION
```